gtk/docs.md

```diff
@@ -1 +1,502 @@
 <!-- file * -->
+<!-- struct TextBuffer -->
+You may wish to begin by reading the
+[text widget conceptual overview](https://developer.gnome.org/gtk3/stable/TextWidget.html)
+which gives an overview of all the objects and data
+types related to the text widget and how they work together.
+<!-- impl TextBuffer::fn insert_markup -->
+Inserts the text in `markup` at position `iter`. `markup` will be inserted
+in its entirety and must be nul-terminated and valid UTF-8. Emits the
+`TextBuffer::insert-text` signal, possibly multiple times; insertion
+actually occurs in the default handler for the signal. `iter` will point
+to the end of the inserted text on return.
+
+Since: 3.16
+
+## `iter`
+location to insert the markup
+## `markup`
+a nul-terminated UTF-8 string containing [Pango markup](https://developer.gnome.org/pygtk/stable/pango-markup-language.html)
+## `len`
+length of `markup` in bytes, or -1
+<!-- struct TextIter -->
+You may wish to begin by reading the
+[text widget conceptual overview](https://developer.gnome.org/gtk3/stable/TextWidget.html)
+which gives an overview of all the objects and data
+types related to the text widget and how they work together.
+<!-- struct Container -->
+A GTK+ user interface is constructed by nesting widgets inside widgets.
+Container widgets are the inner nodes in the resulting tree of widgets:
+they contain other widgets. So, for example, you might have a `Window`
+containing a `Frame` containing a `Label`. If you wanted an image instead
+of a textual label inside the frame, you might replace the `Label` widget
+with a `Image` widget.
+
+There are two major kinds of container widgets in GTK+. Both are subclasses
+of the abstract `Container` base class.
+
+The first type of container widget has a single child widget and derives
+from `Bin`. These containers are decorators, which
+add some kind of functionality to the child. For example, a `Button` makes
+its child into a clickable button; a `Frame` draws a frame around its child
+and a `Window` places its child widget inside a top-level window.
+
+The second type of container can have more than one child; its purpose is to
+manage layout. This means that these containers assign
+sizes and positions to their children. For example, a `HBox` arranges its
+children in a horizontal row, and a `Grid` arranges the widgets it contains
+in a two-dimensional grid.
+
+# Height for width geometry management
```

Comment:
It's not strange, the generator pulls everything there is indiscriminately but the actual bindings don't support subclassing so what are you gonna do.